xcpmd/src/acpi-events.c

```diff
@@ -166,64 +439,56 @@ void acpi_events_read(void)
         process_acpi_message(acpi_buffer, len);
 #ifdef XCPMD_DEBUG
         acpi_buffer[len] = '\0';
-        xcpmd_log(LOG_DEBUG, "~ACPI-event: %s\n", acpi_buffer);
+        //xcpmd_log(LOG_DEBUG, "~ACPI-event: %s\n", acpi_buffer);
```

Comment:
The answer for my particular use-case was "rarely," but I can go ahead and uncomment it.